tests/testfunctions/src/main/java/com/serverless/ApiGatewayResponse.java

```diff
@@ -0,0 +1,129 @@
+package com.serverless;
+
+import java.nio.charset.StandardCharsets;
```

Comment:
Were these code files for the test lambda autogenerated by the serverless plugin?